src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -269,6 +269,9 @@ def remove_group(fs, vol_spec, groupname):
     except cephfs.Error as e:
         if e.args[0] == errno.ENOENT:
             raise VolumeException(-errno.ENOENT, "subvolume group '{0}' does not exist".format(groupname))
+        elif e.args[0] == errno.ENOTEMPTY:
+            raise VolumeException(-errno.ENOTEMPTY, f"cannot delete subvolume group {groupname}, "
+                                  "it is not empty")
```

Comment:
i feel `cannot delete subvolume group {groupname} since it is not empty` looks better